(Ready) Proofreading Mushroom Succubus battle

Giratena opened the pull request.

Giratena committed:
Fixed incorrect translations

Giratena committed:
Further proofreading tests

Giratena commented:
Proofreading a couple of things in the Mycella fight.

@Kyrari - Not bad! Please feel free to check out some of the fixes I made as some of the translations were off the mark. Once you got into the flow though I saw fewer issues, so nice job!

I hope looking at the changes I've made continues to help your translation skills improve ^^

Giratena committed:
More Proofreading

Giratena committed:
Update

Giratena committed:
Update description

Giratena committed:
Minor changes

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -296,7 +296,7 @@ EランクからDランクに昇格するためには、\nクエストを１つ
 ほら、簡単に捕まえちゃった❤パイズリしてほしいからおっきくしてたんでしょ❤それとも仲間がパイズリされてるのを見て羨ましくなっちゃった？,Come on; I caught you so easily～❤ You’re so hard because you wanted a titjob; right?❤ Were you jealous of your ally when he got one?
 えー、じゃあなんでおっきくなってるのかなー❤こうやって挟んでるだけで、おちんちんガチガチになってるのに❤キミも本当は期待してるんでしょ？ほら、おっぱいでぎゅ～❤,No? Then why are you so big already～❤ It got rock hard when I squeezed your dick like this～❤ The truth is; you’re also looking forward to it～❤ Here; I’ll squeeze my breasts together～❤
-このままじっとしてて❤そうすればさっきの人と同じように、とっても気持ち良いパイズリをしてあげるから❤,Just stay like that～❤ And now I’ll pleasure you with my boobs just like the previous person～❤
+このままじっとしてて❤そうすればさっきの人と同じように、とっても気持ち良いパイズリをしてあげるから❤,Just stay like that～❤ And now I’ll pleasure you between my boobs just like the previous person～❤
```
Maybe "just like the last guy" at the end instead? What's there now feels a little stilted.

Giratena replied:
Yep that's a perfect fix, thank you!

HotHeadEmperor commented on CSV/Language/English/Card/10032.txt:
```diff
@@ -56,7 +56,7 @@ EU32_pafupafuAnime,
 あは、腰をガクガクさせちゃってよっぽど気持ちいいんですね。\nそのまま蕩けて、イクまでじっとしていてくださいね❤くすくす❤,Aha~ Your hips are trembling like crazy. You must be feeling good. Stay just like that until you cum; okay? *Giggle*❤
 うふふ❤もうあんまり力が入らないみたいですねぇ。\n胞子に力を吸われて、体がふらふらしちゃうんですか？,Ufufu~ You can't even hold yourself up anymore; can you? Did my spores drain all your strength and leave your body weak and shaky?❤
-ほら、もっといっぱい胞子をつけてあげますね。\nおっぱいで挟んで、ずりずり❤ずりずり❤幸せになっちゃってください❤,Here; let me coat you with even more spores. I'll sandwich your cock between my tits and grind them against you; rubbing; teasing... Let bliss overwhelm you~❤
+ほら、もっといっぱい胞子をつけてあげますね。\nおっぱいで挟んで、ずりずり❤ずりずり❤幸せになっちゃってください❤,Here; let me coat you with even more spores. I'll sandwich your cock between my tits and grind them against you; rubbing; teasing... FAll into bliss~❤
```
Extra capitalization on "Fall"

Giratena replied:
Good spot

HotHeadEmperor commented on CSV/Language/English/Card/10032.txt:
```diff
@@ -181,7 +181,7 @@ EU32_pafupafuAnime,
 そんなこと言われても、止まらないんですよね❤\n腰を振るたびにビリビリって快感が体中を走って、頭の中は\nおっぱいのやわらかさと匂いでいっぱいで、もう天国ですもんね❤,But no matter what I say; you just won't stop; will you?❤ Every thrust sends shivers of pleasure throughout your body; and your mind is filled with the softness and scent of my boobs... It must feel like you're in heaven already.❤
 さあ、さあ、それじゃあエナジーを出してしまいましょう？\nあちらで戦っている仲間のことなんて忘れて、\n気持ちいいお貢ぎ射精をするんです❤,Now then; why don't you release all that mana? Forget about those friends of yours fighting over there; and treat me to your cum with a sweet ejaculation~
-悪い魔物にエナジー出して、おちんちんからびゅっ、びゅーっておもらし❤\nきっと事態はもっと悪くなってしまうと思いますけど、別にいいですよね❤,Release your energy on the nasty monster; and squirt from your cock with a "splurt; splurt"~❤ I think things will get much worse, but that's fine, isn't it~?❤
+悪い魔物にエナジー出して、おちんちんからびゅっ、びゅーっておもらし❤\nきっと事態はもっと悪くなってしまうと思いますけど、別にいいですよね❤,Release your load unto the evil monster; spurting and leaking uncontrollably~❤ It probably won't end well for you; but you don't mind; don't you?❤
```
"you don't mind, do you?"

Giratena replied:
Nice spot!

HotHeadEmperor commented on CSV/Language/English/Card/10031.txt:
```diff
@@ -1,15 +1,19 @@
 [1].このカードが場に出たとき発動する。「よわよわ胞子」を１枚手札に加える。\n([2].戦闘時に相手に「衰弱」(毎ターン終了時に攻撃力を50ポイントダウン,4ターン)を付与する。),[1]. When this card enters the field; search your deck for "Weakening Spores" and add it to your hand.\n([2]. When this card enters combat; inflicts [weakness] on foe ([Weakness] = Attack power is reduced by 50 points for 4 turns.))
+[2].攻撃力がダウンした相手モンスターが破壊されたとき発動する。「<color=#fd7e00><u><link="10031">Mushroom Girl</link></u></color>」を1体特殊召喚し、その攻撃力を破壊されたモンスターの元々の攻撃力の半分だけアップする。,
+([2].戦闘時に相手に「衰弱」(毎ターン終了時に攻撃力を50ポイントダウン,4ターン)を付与する。),([2].Inflicts "Weakness" on foe during battle (reduces attack power by 50 points at the end of each turn; for 4 turns))
+キノコの体を持つ魔物の少女。にこやかな見た目とは裏腹に人間のことをただの食料としか思っておらず、死にそうになって干からびていく男の姿を見ることが好きである。\n\n　彼女の体から放出される胞子は相手に恍惚感を与えると同時にその筋力を吸収する能力を持っている。かわいい見た目に騙されて彼女に抱き着かれてしまえば、屈強な戦士であってもあっという間に子供のように無力にされて、そのままゆっくりと死ぬまで精を吸い取られてしまうだろう。\n　干からびた男はその場に放置することが多いが、気に入った相手には胞子を植え付けて自分の分身を作ることがあり、そのさいには男の遺伝子を吸収してより強力な個体が生まれてくるのだという。,A monster girl with the body of a mushroom. Despite her friendly smile; she thinks of humans  ...
```
Just a line break to remove here.

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -67,36 +67,36 @@ EランクからDランクに昇格するためには、\nクエストを１つ
+ほらほらもっと見てぇ❤\nおっぱいに釘付けになってぇ❤,Hora hora~❤ Just keep staring~❤ Don’t take your eyes off of my breasts~❤
 あううっ…❤,Auuu...❤
-くすくす❤頭の中ぼんやりしているうちに、\nパイズリとかぱふぱふで、たっぷり搾り取ってあげるね❤,Teehee～❤ If you’re going to be that careless; I’m going to squeeze you a lot with my boobs～❤
+くすくす❤頭の中ぼんやりしているうちに、\nパイズリとかぱふぱふで、たっぷり搾り取ってあげるね❤,Teehee～❤ While your mind is all hazy and fuzzy;\nI'll thoroughly drain you dry using my Paizuri and Puff-Puff skills~❤
```
Line break to delete

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -142,8 +142,8 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-あなたのターンからだね！くすくす❤おっぱいに見とれてて大丈夫？,It’s your turn! Teehee～❤ Are you sure you’ll be fine when you’re staring at my boobs?
-ほら、また見ていいよ❤戦う前からおっぱいにメロメロになっちゃえ❤,Come on; stare at them even more～❤ You’ll fall in love with my boobs before the fight has even started～❤
+あなたのターンからだね！くすくす❤おっぱいに見とれてて大丈夫？,It’s your turn! Teehee～❤ Are you sure you’ll be ok fighting with my boobs constantly in your field of view?
+ほら、また見ていいよ❤戦う前からおっぱいにメロメロになっちゃえ❤,Hora~ You can stare at them again if you like~❤ Go on and fall in love with my boobs before we start fighting~❤
```
I feel like 'even' was pretty important here

Giratena replied:
The original Japanese actually didn't mention the word "even" so I'll leave it as is.

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -160,53 +160,53 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-あは、押し倒されているのに抵抗よわーい♪サキュバスに気持ちいいことされるって思ったら体動かなくなっちゃったのかなぁ❤,Ehe～❤ Even though I’m pushing you down; you’re barely resisting it～♪ I wonder if your body isn’t moving because you’re thinking of how good it'll feel to be pleasured by a succubus～❤
-挿入はしないで腰をこすりつけるだけでイかせてあげる❤童貞のまま射精して、死んじゃえ❤死んじゃえ❤くすくす❤,I’m not going to insert it. I'll make you cum from just my legs～❤ If a virgin cums from this; they’re gonna die～❤ Teehee～❤
-ほらほら、上にまたがって素股でいじめてあげる❤こうやって腰をくねらせてっと…❤,Hora; hora～❤ I’ll ride you and tease you with my thighs～❤ I’ll move my thighs just like this...❤
+騎乗位みたいに上にまたがって搾り取ってあげる❤ほら、上に乗っちゃうよ？,I'll straddle your hips and milk you cock dry~❤ Look; I'm getting on top of you now~
```
'your cock.' also the last line feels rigid. Make something like 'I'll climb on top now' or something

Giratena replied:
Good call. I'll fix that.

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -160,53 +160,53 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-ほらほら、上にまたがって素股でいじめてあげる❤こうやって腰をくねらせてっと…❤,Hora; hora～❤ I’ll ride you and tease you with my thighs～❤ I’ll move my thighs just like this...❤
+騎乗位みたいに上にまたがって搾り取ってあげる❤ほら、上に乗っちゃうよ？,I'll straddle your hips and milk you cock dry~❤ Look; I'm getting on top of you now~
+あは、押し倒されているのに抵抗よわーい♪サキュバスに気持ちいいことされるって思ったら体動かなくなっちゃったのかなぁ❤,Ehe～❤ Even though I’m pushing you down; you’re barely resisting it~♪ Maybe it's because you’re thinking of how good it'll feel to be pleasured by a succubus?~❤
+挿入はしないで腰をこすりつけるだけでイかせてあげる❤童貞のまま射精して、死んじゃえ❤死んじゃえ❤くすくす❤,But I'm not going to put your dick in my honey pot. I'll just make you cum by grinding my hips against you~❤ You're gonna pass away ejaculating while still being a virgin~ Die~❤ Die!~❤ Teehee~❤
```
Honey pot feels weird. Maybe just 'put it in' or something like that

Giratena replied:
I feel like it fits the character more. But it's a good comment. I'll leave it as is.

HotHeadEmperor commented on CSV/Language/English/Card/10031.txt:
```diff
@@ -3,36 +3,36 @@ default
+あ、でもわたしに手でしこしこされると\nあなたの体にわたしの胞子がついちゃうかも。,Oh; though if I stroke you with my hand; my spores might get on your body...
+胞子がつくと生命力をどんどん吸い取られると思うけど…。\n気持ちいいからいっか❤しこしこしちゃうねー❤,If my spores get on you; they could steadily drain your life force... But it will make you feel good; so who cares; right?~❤ I'll start stroking you nice and slow~❤
+手コキしてあげる❤おちんちんをしこしこー❤,I'll give you a handjob~❤ Stroking your cock up and down~❤
+んふふ、普通にしこしこされるより気持ちいい？\nわたしの体から出た胞子が体について、\nあなたの力を奪っているんだよ❤,Hehe; does it feel better than regular handjob? The spores from my body are getting on your dick and sapping your strength~❤
```
Better than 'a' regular handjob

Giratena replied:
Good call

HotHeadEmperor commented on CSV/Language/English/Card/10031.txt:
```diff
@@ -108,15 +110,15 @@ EU31_paizuriAnime,
-っていってもおまんこの中じゃなくて、\nパンツにこすりつけてるだけだけど❤\nくすくす❤,But it’s not inside my pussy; just rubbing against my panties~❤ *Giggle*❤
-ほら、ぎゅ～ってしながら喘いであげる❤\nあん❤あん❤あぁん❤いっぱい出してぇ❤\nスカートの中に、あ～ん❤あなたのエナジーちょうだぁい❤,Come on; squeeze tight and moan~❤ Ah~❤ Ah~❤ Ah~❤ Keep releasing.❤ I want your mana in my skirt~❤
-やぁん❤どぴゅどぴゅ～ってたくさん出てる❤\nエナジーおいしい～❤,Yay~❤ You’re releasing so much~❤ Your mana is delicious~❤
+胞子まみれになって、\nわたしのことはらませてぇ❤\n胞子にかけたら子どもができちゃう～❤きゃはは❤,Cover me in your spores and make me pregnant~❤ If you cover my panties with your spores; I might end up pregnant with your child~❤ Kyahaha~❤
```
Id give this another look. Maybe seed instead of spores? Because it makes the MC sound like a mushroom too if she's saying he has his own spores.

Giratena replied:
That is a good call. The Japanese makes it sound that way but I'll reword it. Nice spot!

HotHeadEmperor commented on CSV/Language/English/Card/10031.txt:
```diff
@@ -108,15 +110,15 @@ EU31_paizuriAnime,
-やぁん❤どぴゅどぴゅ～ってたくさん出てる❤\nエナジーおいしい～❤,Yay~❤ You’re releasing so much~❤ Your mana is delicious~❤
+胞子まみれになって、\nわたしのことはらませてぇ❤\n胞子にかけたら子どもができちゃう～❤きゃはは❤,Cover me in your spores and make me pregnant~❤ If you cover my panties with your spores; I might end up pregnant with your child~❤ Kyahaha~❤
+最後はおもいっきり抱きしめてあげるね❤\nあなたも深く腰を突きさすの❤\nいちばん深ーいところで中出しだよ❤,In the end; I’ll hug you tightly.❤ You should thrust your hips deep inside me too~❤ Thrust into my entrance and cum when you reach the deepest part~❤
+っていってもおまんこの中じゃなくて、\nパンツにこすりつけてるだけだけど❤\nくすくす❤,...thrust deeply while you're rubbing against my panties; of course~❤ *Giggle*❤
```
Capital here

HotHeadEmperor commented on CSV/Language/English/Card/10031.txt:
```diff
@@ -108,15 +110,15 @@ EU31_paizuriAnime,
+最後はおもいっきり抱きしめてあげるね❤\nあなたも深く腰を突きさすの❤\nいちばん深ーいところで中出しだよ❤,In the end; I’ll hug you tightly.❤ You should thrust your hips deep inside me too~❤ Thrust into my entrance and cum when you reach the deepest part~❤
+っていってもおまんこの中じゃなくて、\nパンツにこすりつけてるだけだけど❤\nくすくす❤,...thrust deeply while you're rubbing against my panties; of course~❤ *Giggle*❤
+ほら、ぎゅ～ってしながら喘いであげる❤\nあん❤あん❤あぁん❤いっぱい出してぇ❤\nスカートの中に、あ～ん❤あなたのエナジーちょうだぁい❤,Come on; squeeze tight and moan~❤ Ah~❤ Ah~❤ Ah~❤ Release your sperm~❤ I want your mana in my skirt~❤
+やぁん❤どぴゅどぴゅ～ってたくさん出てる❤\nエナジーおいしい～❤,Aaahn~❤ Your cock is spurting out so much with those thick; hot jets❤ Mmm; your energy is so delicious~❤
```
Maybe pulses instead of jets

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -2,10 +2,12 @@ default
+村娘に化けていた下級サキュバス。キノコのカードを好んで使う。純真そうな見た目でありながら、相手の能力を下げて無力化し一方的にいたぶるのが好きというサディスティックな一面を持つ。胸が体格の割に大きく、パイズリが得意。,Mycella. A lower-class succubus disguised as a village girl. Prefers to use mushroom cards. She looks innocent; but holds a sadistic side that enjoys lowering her opponent’s abilities and bullying them one-sidedly. She has quite big breasts and is skilled at giving titjobs.
 下級サキュバスとして一見平凡な能力に見えるが魔力を隠すのが得意であり、たいていのカードマスターには見抜かれない擬装ができる。エルルは魔力観察については上位ランクのマスター並みの実力があるため、今回は擬装がバレてしまった。もし何も気が付かなければ、完全に油断したところをいつの間にかキノコの力で弱体化させられて、一方的に精を搾り取られてしまっていただろう。,Although her abilities might seem average for a lower-class succubus; she’s skilled at hiding her own magical power so that your average cardmasters cannot see it. She was exposed by Eruru because she had magic observatio ...
-村娘に化けていた下級サキュバス。キノコのカードを好んで使う。純真そうな見た目でありながら、相手の能力を下げて無力化し一方的にいたぶるのが好きというサディスティックな一面を持つ。胸が体格の割に大きく、パイズリが得意。\n下級サキュバスとして一見平凡な能力に見えるが魔力を隠すのが得意であり、たいていのカードマスターには見抜かれない擬装ができる。エルルは魔力観察については上位ランクのマスター並みの実力があるため、今回は擬装がバレてしまった。もし何も気が付かなければ、完全に油断したところをいつの間にかキノコの力で弱体化させられて、一方的に精を搾り取られてしまっていただろう。,A lower-class succubus disguised as a village girl. Prefers to use mushroom cards. She looks innocent; but holds ...
+村娘に化けていた下級サキュバス。キノコのカードを好んで使う。純真そうな見た目でありながら、相手の能力を下げて無力化し一方的にいたぶるのが好きというサディスティックな一面を持つ。胸が体格の割に大きく、パイズリが得意。\n下級サキュバスとして一見平凡な能力に見えるが魔力を隠すのが得意であり、たいていのカードマスターには見抜かれない擬装ができる。エルルは魔力観察については上位ランクのマスター並みの実力があるため、今回は擬装がバレてしまった。もし何も気が付かなければ、完全に油断したところをいつの間にかキノコの力で弱体化させられて、一方的に精を搾り取られてしまっていただろう。,A lower-class succubus disguised as a village girl. Prefers to use mushroom cards. She looks innocent; but has a ...
```
Cardmaster one word after 'lower ranked'

Giratena replied:
Nice

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -73,14 +75,14 @@ EランクからDランクに昇格するためには、\nクエストを１つ
 そうです。わたしサキュバスでーす❤\nこんにちはー❤,As you guessed; I’m a succubus~❤ Good to meet you!~❤
 目の前のサキュバスが羽をあらわにし、僕たちにあからさまに魔力をぶつけてきた。まさかこんな街の近くにまたサキュバスがいるだなんて！,Suddenly; the succubus before our eyes revealed her wings; unleashing her magical power directly at us. To think there would be another succubus so close to the city!
-ギルドで言われたように急いで逃げるべきか…。でもこんなに接近していたらそう簡単には逃げられない…。,Should we hurry and run away as they told us at the guild...? But we're already right in front of her; we might not be able to get away so easily...
+ギルドで言われたように急いで逃げるべきか…。でもこんなに接近していたらそう簡単には逃げられない…。,Should we do what as the guild said and quickly run away...? But she's already right in front of us. We might not be able to get away easily...
```
Remove as

Giratena replied:
Good call

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -200,33 +202,33 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-スカートの中に興味ある？ほら、ぴらってめくって見せたりしてぇー❤,Are you curious about what’s under my skirt? Come on; flip it and see what’s underneath～❤
-あはは、反応かわいいー❤目が釘付けじゃん❤命がかかった戦いの最中なのにパンツなんかに見とれちゃうんだ❤,Ahaha; your reactions are so cute～❤ You can’t take your eyes off of them～❤ Even though we’re fighting for our lives; you’re getting captivated by my panties～❤
-ほら、こっちおいで～❤おちんちん、ここにたくさんこすりつけていいよ❤パンツに向かって腰ふりへこへこしてぇ、気持ちよくなろ？くすくす❤,Come closer～❤ Your dick is going to get rubbed a lot over here～❤ Thrust your hips against my panties. It’ll feel good～❤ Teehee～❤  
+スカートの中に興味ある？ほら、ぴらってめくって見せたりしてぇー❤,Are you interested in what's under my skirt? Look; I might just flip it up and show you a little peek~❤
```
Maybe 'give you a little peek' here

Giratena replied:
Also good

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -200,33 +202,33 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-あはは、反応かわいいー❤目が釘付けじゃん❤命がかかった戦いの最中なのにパンツなんかに見とれちゃうんだ❤,Ahaha; your reactions are so cute～❤ You can’t take your eyes off of them～❤ Even though we’re fighting for our lives; you’re getting captivated by my panties～❤
-ほら、こっちおいで～❤おちんちん、ここにたくさんこすりつけていいよ❤パンツに向かって腰ふりへこへこしてぇ、気持ちよくなろ？くすくす❤,Come closer～❤ Your dick is going to get rubbed a lot over here～❤ Thrust your hips against my panties. It’ll feel good～❤ Teehee～❤  
+スカートの中に興味ある？ほら、ぴらってめくって見せたりしてぇー❤,Are you interested in what's under my skirt? Look; I might just flip it up and show you a little peek~❤
+あはは、反応かわいいー❤目が釘付けじゃん❤命がかかった戦いの最中なのにパンツなんかに見とれちゃうんだ❤,Ahaha; your reactions are so cute～❤ You can’t take your eyes off of them～❤ ven in the middle of a life-or-death battle; you're getting distracted by panties~❤
```
Missing E in even

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -200,33 +202,33 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-あは、あっという間におちんちんガチガチにされちゃったねー❤サキュバス相手にエッチのまねごと❤こんなので勃起しちゃうんだー❤,Ehe～❤ Before I realized it; your dick was already this hard～❤ What we’re doing is just pretend sex～❤ And yet you’re already so hard～❤
-本番なんてしなくてもおちんちんこすってあげれば楽勝みたい❤ほおら、あなたのマスターのことなんて、わたしのテクニックで忘れさせてあげる❤くすくす❤,Looks like your dick’s becoming satisfied so easily that I don't even have to insert it～❤ Here; I’ll make you forget about your master with my techniques～❤ Teehee～❤
+あは、あっという間におちんちんガチガチにされちゃったねー❤サキュバス相手にエッチのまねごと❤こんなので勃起しちゃうんだー❤,Ehe～❤ Before I realized it; your dick was already this hard～❤ What we’re doing is just pretend sex～❤ And yet you’re already so hard~❤
+本番なんてしなくてもおちんちんこすってあげれば楽勝みたい❤ほおら、あなたのマスターのことなんて、わたしのテクニックで忘れさせてあげる❤くすくす❤,Looks like your dick’s getting satisfied so easily that I don't even have to insert it into my honey pot~❤ Here; I’ll use my skills to make you forget about your master~❤ Teehee~❤
```
Honey pot still sounds weird

Giratena replied:
Hmm I think it fits. Will leave as is

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -200,33 +202,33 @@ EランクからDランクに昇格するためには、\nクエストを１つ
 がっつり谷間見せつけちゃう❤あは、目が釘付けー♪かわいいじゃん❤マスターくんはこのおっぱいでパイズリしてあげたらどんな声で鳴くのかなぁ❤くすくす❤,I’m showing off my cleavage～❤ Ehe; your eyes are completely glued to it～♪ So cute～❤ I wonder what kind of moans you’ll make when I give you a titjob with these boobs～❤ Teehee～❤
-ねえ見て見て？服の上からでもわかる大きなおっぱい❤これを今からサービスでぇ…❤,Hey; are looking at my breasts? I’m sure you can tell how big they are; even through my clothes～❤ I’ll give you a service with them now...❤
-がっつり谷間見せつけちゃう❤あは、目が釘付けー♪キミはこのおっぱいでパイズリしてあげたらどんな声で鳴くのかなぁ❤楽しみー❤,I’m showing off my cleavage～❤ Ehe; your eyes are completely glued to it～♪ I wonder what kind of moans you’ll make when I give a titjob with these boobs～❤ I can’t wait～❤
+ねえ見て見て？服の上からでもわかる大きなおっぱい❤これを今からサービスでぇ…❤,Hey; are looking at my breasts? I’m sure you can tell how big they are; even through my clothes~❤ Maybe I do a little something for you...❤
```
I'll instead of I

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -200,33 +202,33 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-ねえ見て見て？服の上からでもわかる大きなおっぱい❤これを今からサービスでぇ…❤,Hey; are looking at my breasts? I’m sure you can tell how big they are; even through my clothes～❤ I’ll give you a service with them now...❤
-がっつり谷間見せつけちゃう❤あは、目が釘付けー♪キミはこのおっぱいでパイズリしてあげたらどんな声で鳴くのかなぁ❤楽しみー❤,I’m showing off my cleavage～❤ Ehe; your eyes are completely glued to it～♪ I wonder what kind of moans you’ll make when I give a titjob with these boobs～❤ I can’t wait～❤
+ねえ見て見て？服の上からでもわかる大きなおっぱい❤これを今からサービスでぇ…❤,Hey; are looking at my breasts? I’m sure you can tell how big they are; even through my clothes~❤ Maybe I do a little something for you...❤
+がっつり谷間見せつけちゃう❤あは、目が釘付けー♪キミはこのおっぱいでパイズリしてあげたらどんな声で鳴くのかなぁ❤楽しみー❤,You can clearly see my deep cleavage in full view now~❤ Ehe; your eyes are completely glued to them~♪ I wonder what kind of moans you’ll make when these breasts are around your dick?~❤ I can’t wait～❤
```
'cleavage' is singular, so 'them' either needs to be changed to 'it' or 'my breasts' or 'my chest'

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
+だめっ、早く正気に戻って！その手を止めるのよ！くっ！,No; snap out of it! Stop your moving hands! ..ack!
+あらら、よそ見しているからスライムちゃんの攻撃をくらっちゃった。お仲間のマスターさん、かわいそー❤,Ara; ara. She looked away and got hit by the slime succubus. I feel bad for your cardmaster friend~❤
+くすくす❤それでも手止まらないんだ。おっぱい見ながらシコシコ気持ちいい❤夢中になりすぎて仲間の声も聞こえてない❤,Teehee～❤ And yet; your hands aren't stopping at all. Jerking off while staring at my boobs feels too good～❤ You’re so absorbed in it you can’t even listen to what your friend is saying~❤
+特別サービスだよ❤谷間に集中して、じっと見つめて❤ほらほらほら❤,This is a special service after all~❤ Look at my cleavage and focus closely as a jiggle my tits back and forth~❤ Hora; hora~❤
```
'as I jiggle'

Giratena replied:
Nice one

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-って、もうどうせ射精しちゃうか❤ほら、谷間にだしていいよー❤マスターくんのエナジー、わたしのおっぱいに捧げちゃお❤,Oh; I think you’re about to cum～❤ Here; you can cum on my cleavage～❤ Offer all your energy to my breasts～❤
-ほら❤谷間をゆさゆさゆさ❤ちゃ～んと谷間を狙って❤精液、だして❤,Hora; hora～❤ My cleavage is swaying～❤ Make sure to aim at them～❤ Let out your semen～❤
+あはは、よわよわおちんちん気持ちいいね❤早くシコシコしないと谷間が見えなくなっちゃうぞー❤,Ahaha; your weak little cock feels good; doesn't it?❤ You better stroke it faster or you won't get to see my cleavage anymore~❤
+って、もうどうせ射精しちゃうか❤ほら、谷間にだしていいよー❤マスターくんのエナジー、わたしのおっぱいに捧げちゃお❤,Oh; you’re about to cum?~❤ Here; you can shoot all over my boobs~❤ Offer all your energy to my breasts～❤
```
'shoot it all over' might sound nicer

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
 はひぃっ…❤,Hahieee...❤
-あはは❤すごい量❤後ろで仲間が戦ってるのに、裏切りオナニー❤気持ち良かった？マスターくん❤,Ahaha～❤ What a huge load～❤ Even though your friend is fighting back there; you’re masturbating and betraying her～❤ Did it feel good? Mr. Cardmaster～❤
-谷間を見せてるだけなのに、勝手にシコシコしてエナジーを捧げてくれるなんて、本当にチョロい❤,Even though I just showed you my cleavage; you jerked off to your opponent and offered your energy～❤ You really are a pushover～❤
+あはは❤すごい量❤後ろで仲間が戦ってるのに、裏切りオナニー❤気持ち良かった？マスターくん❤,Ahaha～❤ What a huge load～❤ Even while your friend is fighting behind us; you started masturbating like a traitor~❤ Did it feel good; Mr. Cardmaster~?❤
```
The read-me says ~ is considered a punctuation, so I don't know that we should pair it with another punctuation, like a question mark

Giratena replied:
Good call. I'll remove it

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-あはは❤すごい量❤後ろで仲間が戦ってるのに、裏切りオナニー❤気持ち良かった？マスターくん❤,Ahaha～❤ What a huge load～❤ Even though your friend is fighting back there; you’re masturbating and betraying her～❤ Did it feel good? Mr. Cardmaster～❤
-谷間を見せてるだけなのに、勝手にシコシコしてエナジーを捧げてくれるなんて、本当にチョロい❤,Even though I just showed you my cleavage; you jerked off to your opponent and offered your energy～❤ You really are a pushover～❤
+あはは❤すごい量❤後ろで仲間が戦ってるのに、裏切りオナニー❤気持ち良かった？マスターくん❤,Ahaha～❤ What a huge load～❤ Even while your friend is fighting behind us; you started masturbating like a traitor~❤ Did it feel good; Mr. Cardmaster~?❤
+谷間を見せてるだけなのに、勝手にシコシコしてエナジーを捧げてくれるなんて、本当にチョロい❤,I only showed you my cleavage; and yet you ended up stroking yourself and offered up your energy to me... you really are a pushover~❤
```
Capitalize 'You' since it's a new thought.

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-ふたりのサキュバスに挟み撃ちをされてしまったエルルに成すすべはなく、あっという間に組み伏せられてしまう。,Attacked on both sides by two succubi; Eruru is at a complete loss and held down by them in no time.
-必死の抵抗を試みるエルルだったが、スライム使いのサキュバスに唇を奪われ、おとなしくなってしまった。彼女がどうなってしまうのか。僕にはわからない。,She tried to desperately resist; but the slime-using succubus stole a kiss from her and she quickly turned silent afterwards. I don’t know what will happen to her.
+ふたりのサキュバスに挟み撃ちをされてしまったエルルに成すすべはなく、あっという間に組み伏せられてしまう。,Attacked on both sides by two succubi; Eruru was easily defeated and held down by them in no time.
+必死の抵抗を試みるエルルだったが、スライム使いのサキュバスに唇を奪われ、おとなしくなってしまった。彼女がどうなってしまうのか。僕にはわからない。,She tried to desperately resist; but the slime-using succubus kissed her on the lips; and she quickly turned silent. What would happen to her after that; I don't know.
```
Maybe ' I don't know what happened to get after that'

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-罪悪感があってもさぁ、おちんちんおっぱいでズリズリされたら、気持ちよくて気持ちよくてどうにもならないんだよね❤,Even though you’re feeling guilty; you still can't help but enjoy it when your dick gets squeezed by my breasts～❤
-わたしのおっぱい❤とっても柔らかいもんね❤この前挟んであげた村の男の人もすぐにあへぇ❤ってなさけなーい喘ぎ声あげちゃったけど、マスターくんも同じ❤,My breasts are very soft; aren’t they? The previous villagers I squeezed with these breasts also immediately went “ahiee～❤”; just the same as you～❤
-頑張って修行してきたのに普通の人と変わらないんだよ？くすくす❤今までの努力、みーんな無駄だったのかもねー❤,Even though you’ve trained really hard; in the end you’re no different than your average villager. Teehee～❤ Maybe all your hard work up to now was pointless～❤
+あーあ、マスターくんの顔とろけちゃった❤ちょっとパイズリされただけでとろけっちゃうよわよわおちんちんで、よくサキュバスと戦おうと思ったね？,SAww; the cardmaster’s already melting from the pleasure~❤ Did you really think you could fight a succubus with such a weak dick that gets like this from the slightest touch of my breasts?
```
Accidental S here

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-わたしのおっぱい❤とっても柔らかいもんね❤この前挟んであげた村の男の人もすぐにあへぇ❤ってなさけなーい喘ぎ声あげちゃったけど、マスターくんも同じ❤,My breasts are very soft; aren’t they? The previous villagers I squeezed with these breasts also immediately went “ahiee～❤”; just the same as you～❤
-頑張って修行してきたのに普通の人と変わらないんだよ？くすくす❤今までの努力、みーんな無駄だったのかもねー❤,Even though you’ve trained really hard; in the end you’re no different than your average villager. Teehee～❤ Maybe all your hard work up to now was pointless～❤
+あーあ、マスターくんの顔とろけちゃった❤ちょっとパイズリされただけでとろけっちゃうよわよわおちんちんで、よくサキュバスと戦おうと思ったね？,SAww; the cardmaster’s already melting from the pleasure~❤ Did you really think you could fight a succubus with such a weak dick that gets like this from the slightest touch of my breasts?
+仲間の女の子を捕まえたわるーいサキュバス相手なのに、あん❤あん❤って喘ぐことしかできずに、腰がびくびくしちゃってる。もっとパイズリされたいんだよね？なさけなーい❤,You're facing a wicked succubus who's captured your female companion; yet all you can do is moan "Ahhh~❤ Ahhh~❤" while your hips tremble uncontrollably. You really want your cock massaged between my tits; don't you? How pathetic❤
```
Punctuation before the last heart. maybe a ~

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
+ボクのちょろマゾおちんちんのせいですって❤ごめんなさいって❤ほーら、早くあの子に謝って❤,Come on; hurry up and apologize to her~❤ Tell her; “It’s because I have a pushover; masochist dick~❤ I’m so sorry~❤” 
+はううっ❤、ごめっ❤エルルさっ❤ごめんなさい・・・❤,Hauuu～❤ I...❤ Eruru...❤ I’m sorry...❤
+あはは、マスターくん。泣きそうな顔になってる❤いまさら罪悪感でも感じてるのかな？もう手遅れなのに。,Ahaha; Cardmaster; are you bawling your eyes out?❤ Are you starting to feel guilty now? But it’s too late for that.
+精液ってねぇ、心の中がぐちゃぐちゃの時の方がおいしいんだよ？いまのマスターくんの精液はとってもおいしそう❤,Did you know that your semen tastes much better when you're in deep anguish? And right now your mana right now is looking especially delicious~❤
```
Remove the extra 'right now your' after 'and'

Giratena replied:
Good call

HotHeadEmperor commented on CSV/Language/English/Card/70011.txt:
```diff
@@ -305,85 +307,85 @@ EランクからDランクに昇格するためには、\nクエストを１つ
-わたしってとってもやさしいからさー、きもちよーく射精できるるように思いっきりぎゅ～ってしてあげるね❤,I’m a very kind succubus～ So I’ll make sure to squeeze your dick and make that ejaculation feel so good you’ll never ever forget it～❤
-マスターくんも自分で腰、突き上げて・・・おっぱいの中に全部だして❤ほら、イケ❤マーゾ❤,Come on; thrust your hips... Let it all out in my breasts～❤ Hora; hora～❤ Cum; you masochist～❤
-あへぇ❤❤あっ、でる、あああっ・・・❤,Ahnn～❤❤ I’m cumming; ahh...❤
+あーあ、完全に顔がとろけちゃった❤おっぱいに降伏❤仲間も裏切って、マスターとしての頑張りも捨てて、へこへこパイズリおねだり❤,Aww; you’ve been completely charmed～❤ Surrendered completely to my breasts～❤ Betrayed your ally; and threw away all your hard work just to beg and thrust your dick between my boobs～❤
```
I think the 'you' you removed here sounded better

Giratena committed:
Update

WisDZ committed:
Partial Fix - Mushroom Girl Card Ability Text

Adding this line fixes the ability text in Easy, but I wasn't able to get the Hard or Ero ability text to function, so I left the text as-is.

Giratena committed:
Hothead proofreading updates

Giratena committed:
Merge branch 'proofreading/mushroom-succubus-70011' of https://github.com/Giratena/Succubus-Duel-English-Translation into proofreading/mushroom-succubus-70011
